KMDefaultSettings.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  KmDefaultSettingPreference.swift
+//  Kommunicate
+//
+//  Created by sathyan elangovan on 03/06/22.
+//
+
+import Foundation
+
+/// `KMDefaultSettings` is used for handling the default conersation assignee,bot,agent settings and storing the data
+public struct KMDefaultSettings {
+    private let defaultBotIds = "DEFAULT_BOT_IDS"
+    private let defaultAgentIds = "DEFAULT_AGENT_IDS"
+    private let defaultAssignee = "DEFAULT_ASSIGNEE"
+    private let defaultTeam = "DEFAUT_TEAM"
+    private let defaultSkipRouting = "DEFAULT_SKIP_ROUTING"
+    
+    private let userDefaults = UserDefaults.standard
+    
+    public init() {}
+    
+    public func setDefaultBotIds(_ botIds : [String]) {
```

Comment:
Use var's `get` and `set` instead of creating a separate getter and setter methods.

Reply:
used `get` & `set` method